src/musikcore/audio/PlaybackService.cpp

```diff
@@ -936,6 +936,9 @@ void PlaybackService::OnStreamEvent(StreamState eventType, std::string uri) {
 
 void PlaybackService::OnPlaybackEvent(PlaybackState eventType) {
     POST(this, MESSAGE_PLAYBACK_EVENT, (int64_t) eventType, 0);
+    if (eventType == PlaybackState::Paused || eventType == PlaybackState::Playing) {
```

Comment:
I don't think this should be necessary any more -- as far as I know the existing event is working as expected. Do you see it not emitting an event on state change without your modification? :)

Reply:
You're right, @lcangen, this is unnecessary as the `transport_playing_format_playing` and `transport_playing_format_paused` appear promptly without it. Great find! :)